Mathlib/CategoryTheory/Filtered/Basic.lean

```diff
@@ -81,7 +81,7 @@ class IsFilteredOrEmpty : Prop where
 3. there exists some object. -/
 @[stacks 002V "They also define a diagram being filtered."]
 class IsFiltered : Prop extends IsFilteredOrEmpty C where
-  /-- a filtered category must be non empty -/
+  /-- a filtered category must be nonempty -/
```

Comment:
I personally prefer non-empty; this may be a matter of taste. Local style could be a good arbiter.

Reply:
See general comment below.